Source/Utils/WarpXTagging.cpp

```diff
@@ -45,7 +45,16 @@ WarpX::ErrorEst (int lev, TagBoxArray& tags, Real /*time*/, int /*ngrow*/)
             const RealVect pos {AMREX_D_DECL((i+0.5_rt)*dx[0]+problo[0],
                                        (j+0.5_rt)*dx[1]+problo[1],
                                        (k+0.5_rt)*dx[2]+problo[2])};
```

Comment:
I might have done this differently, to do the check between the fine tag and parser here, something like this (though this isn't complete).
```
bool tag;
if (ref_patch_parser) {
    tag = (ref_parser(pos[0], pos[1], pos[2]) == 1);
} else {
    tag = (pos > ftlo && pos < fthi);
}
```
This would avoid the somewhat messy definitions of ref_patch_function below.

Reply:
Yes I will fix this

Reply:
@dpgrote fixed this

Reply:
Much nicer!

Reply:
Thanks for the suggestions!